index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <title>Melange for React Developers</title>
-    <script type="module" src="main.js"></script>
+    <script type="module" src="./_build/default/output/Index.js"></script>
```

Comment:
Another approach to avoid the full path to the JS file could be to have melange copy the index.html file into build using `runtime_deps`, and somehow tell vite to load _that_ index.html instead. I think it only makes sense if more assets were added to the html file, because otherwise we'd be exchanging one build path (the js file) with another (the html file).